### PR TITLE
fix: readme link correction

### PR DESCRIPTION
**Pull request by revolunet**
Hello, looks like GitHub changed their search API, this should fix the link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,8 +68,7 @@ Finding projects depends on how the search API is structured for every hosting
 platform. For example, you can find all `publiccode.yml` on GitHub files by
 searching using the frontend or the API.
 
-* [GitHub Search `filename:publiccode.yml
-  path:/`](https://github.com/search?q=filename%3Apubliccode.yml+path%3A%2F)
+* [GitHub Search `path:/publiccode.yml`](https://github.com/search?q=path%3A%2Fpubliccode.yml&type=code)
```
Comment (bvhme) [inline README.md:71]: Great catch. This however is not the right result (15 results). For that I think we might need to use a RegEx for searching. What about this: `path:/(^|\/)publiccode\.yml$/`.

https://github.com/search?q=path%3A%2F%28%5E%7C%5C%2F%29publiccode%5C.yml%24%2F&type=code

```suggestion
* [GitHub Search `path:/(^|\/)publiccode\.yml$/`](https://github.com/search?q=path%3A%2F%28%5E%7C%5C%2F%29publiccode%5C.yml%24%2F&type=code)
```

**Review by bvhme — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -68,8 +68,7 @@ Finding projects depends on how the search API is structured for every hosting
 platform. For example, you can find all `publiccode.yml` on GitHub files by
 searching using the frontend or the API.
 
-* [GitHub Search `filename:publiccode.yml
-  path:/`](https://github.com/search?q=filename%3Apubliccode.yml+path%3A%2F)
+* [GitHub Search `path:/(^|\/)publiccode\.yml$/`](https://github.com/search?q=path%3A%2F%28%5E%7C%5C%2F%29publiccode%5C.yml%24%2F&type=code)
```
Comment (bfabio) [inline README.md:71]: Thanks @revolunet!

We can also go the simple route and do this, wdyt? cc @bvhme 

```suggestion
* [GitHub Search `path:publiccode.yml`](https://github.com/search?q=path%3Apubliccode.yml&type=code)
```

Comment (revolunet) [reply README.md:71]: indeed, it has even more results

Comment (bvhme) [reply README.md:71]: It having more results comes from it including files with other capitalisation (`Publiccode.yml`) and files where it is part of the path. Include more things but less aligned with the Standard. I like the more simple solution even though it is less useful in more precise or machine contexts.

**Pull request merged**